modules/porous_flow/doc/content/modules/porous_flow/hysteresis.md

```diff
@@ -256,13 +256,13 @@ A python script that produces plots of hysteretic capillary pressure and relativ
 
 To include hysteretic capillarity in an existing (non-hysteretic) input file, the following changes need to be made.
 
-1. Any capillary-pressure UserObjects, such as [PorousFlowCapillaryPressureVG](PorousFlowCapillaryPressureVG.md), may be removed from the input file
+1. Any capillary-pressure UserObjects, such as [PorousFlowCapillaryPressureVG](PorousFlowCapillaryPressureVG.md) will no-longer be used, so should be removed from the input file for clarity.
 2. A [PorousFlowHysteresisOrder](PorousFlowHysteresisOrder.md) Material needs to be included.
 3. A Material that computes the porepressure(s) and saturation(s) needs to be included.
 
    - For 1-phase partially-saturated situations, [PorousFlow1PhaseP](PorousFlow1PhaseP.md) should be removed and replaced by [PorousFlow1PhaseHysP](PorousFlow1PhaseHysP.md).  Note the van Genuchten parameter input is $m$ for the non-hysteretic version, but $n$ for the hysteretic version, where $n = 1/(1 - m)$ and $m = 1 - 1/n$.
-   - For 2-phase situations using the two porepressures as the primary variables, [PorousFlow2PhasePP](PorousFlow2PhasePP.md) should be removed and replaced by TODO.
-   - For 2-phase situations using the liquid porepressure and gas saturation as the primary variables, [PorousFlow2PhasePS](PorousFlow2PhasePS.md) should be removed and replaced by TODO.
+   - For 2-phase situations using the two porepressures as the primary variables, [PorousFlow2PhasePP](PorousFlow2PhasePP.md) should be removed and replaced by [PorousFlow2PhaseHysPP](PorousFlow2PhaseHysPP.md).
```

Comment:
Just FYI, you can use `[PorousFlow2PhaseHysPP.md]` and it will automatically use the title heading of the page in the text.

Reply:
Thanks for that tip, @aeslaughter ,

andy